uaclient/contract.py

```diff
@@ -246,6 +244,92 @@ def get_contract_token_for_account(contract_client, macaroon, account_id):
     return contract_token_response['contractToken']
 
 
+def process_repo_entitlement_deltas(orig_access, deltas) -> None:
+    """Process repo-based entitlement deltas.
+
+    :param orig_access: Dict with original entitlement access details before
+        refresh deltas.
+    :param deltas: Dict containing discovered entitlement access deltas.
+    """
+    from uaclient import entitlements
```

Comment:
These imports and the `issubclass` check below suggest to me that this should be a method on `RepoEntitlement`; does that make sense?

Reply:
I think this is comment is still outstanding